### ISSUE TITLE
RawField incompatibility inside of Batch

### ISSUE THREAD

**Issue by mfuntowicz** [pytorch/text#460]
I've updated torchtext to its latest master, and it seems that the setup for Squad I was using before is not working anymore.

More precisely, I was using 2 RawField which allow me to get back from the predicted tokens span to the initial text : 

```python
fields = [
                ('id', RawField()),
                (('content_w', 'content_c'), (text_f, char_f)),
                (('question_w', 'question_c'), (text_f, char_f)),
                ('context', RawField()),
                ('answer', RawField()),
                ('answer_start', LabelField(use_vocab=False)),
                ('answer_end', LabelField(use_vocab=False))
            ]
```
Now, the following set of Field makes the Batch creation crash as : 

```python
  File "/data/Workspace/miniconda3/envs/pytorch/lib/python3.7/site-packages/torchtext/data/iterator.py", line 156, in __iter__
    yield Batch(minibatch, self.dataset, self.device)
  File "/data/Workspace/miniconda3/envs/pytorch/lib/python3.7/site-packages/torchtext/data/batch.py", line 26, in __init__
    self.input_fields = [k for k, v in dataset.fields.items() if
  File "/data/Workspace/miniconda3/envs/pytorch/lib/python3.7/site-packages/torchtext/data/batch.py", line 27, in <listcomp>
    v is not None and not v.is_target]
AttributeError: 'RawField' object has no attribute 'is_target'
```

Is there a new way of handling this ? Or is it possible to make Batch compatible again with RawField ?

_One possible way would be to introduce the property is_target at the RawField level, so that every children of RawField would have this field available._

Thanks for your feedback,
Morgan

**Comment by mttk** [comment]
A fix in #459 has been merged, please try reinstalling from master.

**Comment by mfuntowicz** [comment]
Working now ! Thanks 👍
